src/Task/ApiGen/ApiGen.php

```diff
@@ -30,6 +31,7 @@ class ApiGen extends BaseTask implements CommandInterface
      * @var string
      */
     protected $command;
+    protected $operation = ' generate';
```

Comment:
Maintaining consistency is better than meeting aspirational goals; I think in this case, though, it should still be possible to keep the space out of `$operation`.